File: scala-gatling/src/gatling/scala/org/openapitools/client/model/ResponseTimeFilterFastResult.scala

package org.openapitools.client.model


case class ResponseTimeFilterFastResult (
    _searchId: String,
    _locations: List[ResponseTimeFilterFastLocation],
    _unreachable: List[String]
)
object ResponseTimeFilterFastResult {
    def toStringBody(var_searchId: Object, var_locations: Object, var_unreachable: Object) =
        s"""
        | {
        | "searchId":$var_searchId,"locations":$var_locations,"unreachable":$var_unreachable
        | }
        """.stripMargin
}

File: fsharp-giraffe-server/traveltimeplatform/src/model/ResponseTimeFilterFastResult.fs
namespace traveltimeplatform.Model

open System
open System.Collections.Generic
open traveltimeplatform.Model.ResponseTimeFilterFastLocation

module ResponseTimeFilterFastResult = 

  //#region ResponseTimeFilterFastResult


  type ResponseTimeFilterFastResult = {
    SearchId : string;
    Locations : ResponseTimeFilterFastLocation[];
    Unreachable : string[];
  }
  //#endregion
  